vllm/v1/worker/gpu_model_runner.py

```diff
@@ -223,9 +223,10 @@ def _prepare_inputs(self, scheduler_output: "SchedulerOutput"):
 
         # Get the number of scheduled tokens for each request.
         # TODO: The Python loop can be slow. Optimize.
-        num_scheduled_tokens = []
+        num_scheduled_tokens = np.empty(0, dtype=np.int32)
```

Comment:
Can we keep this a list but just rename it? I'd like to avoid using `append` for numpy arrays.

Reply:
I'll revert this, I think #11111 removed the original error. Great catch :+1: